about.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Page</title>
+  <link rel="stylesheet" href="gallery.html">
+  <link rel="stylesheet" href="index.html">
+  <link rel="stylesheet" href="about.css">
+  <link rel="stylesheet" href="./blog.html">
```

Comment:
these links aren't right. they should be css. 
`<link rel="stylesheet" href="./styles.css">
    <link rel="stylesheet" href="./gallery.css">
    <link rel="stylesheet" href="./about.css">
    <link rel="stylesheet" href="./blog-styles.css">`